### DISCUSSION TITLE
Dudas de Práctica 1 - Requerimientos

### DISCUSSION THREAD

**Discussion by carezus** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#27] [Q&A]
Esta práctica se limita solo a los requerimientos y requerimientos detallados del cliente? O a al llenado completo de la plantilla que viene adjunta que incluye ya temas mas específicos como los requisitos técnicos, la creación de diagrama de negocio, el desarrollo de las fases del proyecto y el diseño de la arquitectuda de solución?

**Comment by Josue-HernandezL** [comment]
tengo la misma duda xd

**Comment by DavidMaralb** [comment]
Pues yo creo que incluye todo, por que es precisamente la fase de identificación de requerimientos, entre mas información puedas recabar y verificar con el cliente, será mejor para el proyecto.

**Comment by lunasaurio** [comment]
Saben si la informacion debe ser inventada o recolectada de algun lugar?

**Reply by Burstniart** [reply]
En el Readme de las prácticas viene la información del caso, a partir de esta se realiza

https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/blob/main/01%20-%20INTRO/practicas/README.md

**Reply by KarenPamelaLopezMedina** [reply]
Muchas gracias!

**Comment by Ingrzs** [comment]
Todo la primera práctica requiere información que tendrías que en la práctica 2,3,4,5 , de esos datos habra que llenarlos, y basate en los datos que estan en la descripción que adjunto el MC

**Comment by lunasaurio** [comment]
Que tal Compañeros, tengo una duda al momento de llenar los requerimientos del documento me topé con la parte de "vistas"
El conflicto que me causa es que ya invesitgue cada una de ellas y aun me cuesta comprender que es lo que quiere, apreciaria sus comentarios.
![image](https://user-images.githubusercontent.com/93229275/155430480-85640b50-ebdc-492c-bbed-440aadd9a561.png)

**Reply by AlexZRamirez** [reply]
Hola lunasaurio, la parte de vistas entra en la etapa de diseño, junto al prototipado (wireframe, diseño de ux/ui) en análisis de requerimientos es definir los requerimientos funcionales y no funcionales, en la etapa previa se realiza una reunión con el cliente, donde preguntas las características del proyecto. Dado que el MC nos dio el caso de estudio para la practica, es cuestión de que desgloses la información que proporciona y plasmarlo en una plantilla de requerimientos, puedes hacerla mas o menos compleja según lo creas conveniente. Retomando tu pregunta, los diagramas UML los utilizarías para representar las interacciones en tu plataforma y como es que va a trabajar la información. En este caso puedes o no utilizar todos los diagramas indicados, estos tienen la única tarea de hacer fluida la comprensión del funcionamiento del proyecto.
